### PR TITLE
proposed CIP001 tweaks for better CIP communication

### PR DESCRIPTION
**Pull request by robby-dermody**

### CODE REVIEW

File: cip-0001.md [hunk 1]
```diff
@@ -173,4 +171,4 @@ The editors don't pass judgement on CIPs. We merely do the administrative & edit
...
\ No newline at end of file
+This document was derived heavily from Bitcoin's [BIP-0001](https://github.com/bitcoin/bips/blob/master/bip-0001.mediawiki), which was derived heavily from Python's PEP-0001. In many places text was simply copied and modified. PEP-0001 text was written by Barry Warsaw, Jeremy Hylton, and David Goodger.  They are not responsible for its use in the Counterparty Improvement Process, and should not be bothered with technical questions specific to Counterparty or the CIP process. Please direct all comments to the [Counterparty forums](https://counterpartytalk.org/c/development) or the [CIP editors](mailto:dweller@devonweller.com).
```
Comment (deweller) [inline cip-0001.md]: I can't determine what changed here!

Comment (robby-dermody) [inline cip-0001.md]: nothing did. EOF was added or removed :shipit:

Comment (jdogresorg) [inline cip-0001.md]: Generally looks good.

> Reasons for denying CIP status include duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, **or not in keeping with the Counterparty philosophy**.

Could possibly use some more clarity as to what the "Counterparty philosophy" is.

Comment (robby-dermody) [inline cip-0001.md]: yeah, that was from the original. let me think how we can improve it (any ideas welcome)

**Comment by robby-dermody** [comment]
first take at addressing @deweller 's points about clarity around Final and Accepted statuses

File: cip-0001.md [hunk 2]
```diff
@@ -33,23 +33,23 @@ Each CIP must have a champion -- someone who writes the CIP using the style and
...
 
 For a CIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
 
-Once a CIP has been accepted, the reference implementation must be completed. When the reference implementation is complete and accepted by the community, the status will be changed to "Final".
+To be accepted, the CIP's pull request must be approved ("ACKed") by at least two project maintainers. A project maintainer is defined as an individual that has commit access to the (counterparty-lib)[https://github.com/CounterpartyXCP/counterparty-lib] repository. If there is only one maintainer, obviously only his or her ACK is necessary.
```
Comment (deweller) [inline cip-0001.md]: This is good.  This is a crucial missing step in our process now.

I'd say change this to "a majority of project maintainers".  That still requires 2 until we get to 4 maintainers.

Comment (robby-dermody) [inline cip-0001.md]: I also tweaked the definition of a maintainer a bit (require a commit in the last 6 months as adam is still listed)... let me know what you think

File: cip-0001.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ Standards Track CIPs consist of two parts, a design document and a reference imp
 
 For a CIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
 
-To be accepted, the CIP's pull request must be approved ("ACKed") by at least two project maintainers. A project maintainer is defined as an individual that has commit access to the (counterparty-lib)[https://github.com/CounterpartyXCP/counterparty-lib] repository. If there is only one maintainer, obviously only his or her ACK is necessary.
+To be accepted, the CIP's pull request must be approved ("ACKed") by a majority of project maintainers. A project maintainer is defined as an individual that has commit access to the (counterparty-lib)[https://github.com/CounterpartyXCP/counterparty-lib] repository, and has made at least one commit in the last six month period. If there is only one maintainer, obviously only his or her ACK is necessary.
```
Comment (deweller) [inline cip-0001.md]: This seems reasonable to me.  We should make the sunset length 1 year to be safe.

File: cip-0001.md (hunk 1 above)
Comment (deweller) [inline cip-0001.md]: How about ...**or not in keeping with the Counterparty philosophy as determined by a majority of the project maintainers.**

Comment (deweller) [inline cip-0001.md]: When you are ready, Robby, let's have you and rubensayshi ACK or noACK this and then I will merge accordingly.

Comment (deweller) [inline cip-0001.md]: @robby-dermody and @rubensayshi - are you ready to merge this?

**Comment by robby-dermody** [comment]
Sounds good will do it later today

**Comment by robby-dermody** [comment]
@deweller I went with 9 months of inactivity instead of 12, for the "project maintainer" definition. Although we generally want to work to only have active or quasi-active folks as project maintainers, I am thinking that 12 is a bit too long (people fall off the map). I am good with 6 or 9, depending on Ruben's views as well here.

if no one else has any other issues I think this is ready to merge, pending @rubensayshi 's feedback and ACK

File: cip-0001.md (hunk 1 above)
Comment (robby-dermody) [inline cip-0001.md]: ACK

**Comment by deweller** [comment]
@rubensayshi - Can you provide feedback, if any or ACK?

**Comment by robby-dermody** [comment]
Merging due to the amount of time this has been outstanding. If any maintainer objections (doubtful), I can revert.

**Comment by deweller** [comment]
ACK

**Pull request merged**